client/signup/steps/social-profiles/social-profiles.tsx

```diff
@@ -13,18 +13,26 @@ import type { ReactElement } from 'react';
 const TextInputWrapper = styled.div`
 	display: flex;
 	align-items: center;
-	border: 1px solid #646970;
-	box-sizing: border-box;
+	gap: 10px;
 	border-radius: 4px;
-
+	border: 1px solid var( --color-neutral-10 );
+	&:focus-within,
+	&:focus-within:hover {
+		border-color: #646970;
+		box-shadow: 0 0 0 2px #e2eaf1;
+	}
+	&:hover {
+		border-color: var( --color-neutral-20 );
+	}
 	.social-logo {
 		opacity: 0.18;
 		margin: 11px 0 11px 10px;
 	}
-	input[type='text'].form-text-input {
+	input.form-text-input {
 		border: none;
-		padding-left: 10px;
```

Comment:
Replaced this with `padding-inline-start: 0px;` and `gap: 10px` in the parent div to handle RTL locales.